From 8297350f1ef4309f4da691765f02409aeadc5a28 Mon Sep 17 00:00:00 2001
From: Jordan Petridis <jpetridis@gnome.org>
Date: Wed, 11 Sep 2024 16:07:09 +0300
Subject: [PATCH] components/pcsc-lite: Install services under system

The autotools build was defaulting to this, but meson
defaults to user.

Revert back to the autotools behavior.

Followup to 26e724cb8cab660047db63d0a2181910cc57dc63
---
 elements/components/pcsc-lite.bst | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/elements/components/pcsc-lite.bst b/elements/components/pcsc-lite.bst
index 1e4655a5d56..ac016864292 100644
--- a/elements/components/pcsc-lite.bst
+++ b/elements/components/pcsc-lite.bst
@@ -10,6 +10,10 @@ depends:
 - components/polkit-gobject.bst
 - components/systemd-libs.bst
 
+variables:
+  meson-local: >-
+    -Dsystemdunit="system"
+
 sources:
 - kind: git_repo
   url: debian_salsa:rousseau/PCSC.git
-- 
GitLab

